tests/mock_tables/config_db.json

```diff
@@ -573,7 +574,8 @@
         "NULL": "NULL"
     },
     "INTERFACE|Ethernet0": {
-        "NULL": "NULL"
+        "NULL": "NULL",
```

Comment:
why do we need NULL:NULL if there is a field (ipv6_use_link_local_only)?

Reply:
Not needed, removed NULL attribute.